app/__init__.py

```diff
@@ -1,11 +1,24 @@
-from flask import Flask
+from flask import Flask, jsonify
 from flask_sqlalchemy import SQLAlchemy
 from flask_cors import CORS, cross_origin
-
+import os
+#create app method
 app = Flask(__name__)
-cors = CORS(app)
+cors = CORS(app, support_credentials=True)
```

Comment:
trying to work in CORS. This is breaking the add_day endpoint currently.

Reply:
Does this mean it's working now with these changes?